content/html/concepts/attributes/terms/class/class.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'Classes'
+
+Description: 'Classes are HTML attributes used to select one or more elements for the application of styling or logic.'
+
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+
+Tags:
+  - 'Attributes'
+  - 'Classes'
+  - 'Style'
```

Comment:
The end of the metadata is missing. There needs to be a CatalogContent: section and a concluding three dashes: `---`

Also, the extra line breaks in the metadata need to be removed.

The complete Metadata should look like this:
```
---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---
```

You can find entries for CatalogContent: [here](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md).